### ISSUE TITLE
Unable to use msg containing arrays of size > 32

### ISSUE THREAD

**Issue by clynamen** [adnanademovic/rosrust#56]
Due to the fact that [arrays of size > 32 do not implement the following traits ](https://doc.rust-lang.org/1.7.0/std/primitive.array.html):

 * Debug
 * Default

It seems not possible to use them with rosrust. This prevents the use of msgs such as nav_msgs/Odometry, where covariances are expressed as [f64, 36]

Would it be ok to define a new type for arrays, with the same semantic, which implements the required traits? I may look into it

**Comment by adnanademovic** [comment]
This will be fixed once the procedural macros based handling of messages is merged into master.
